test/test_autograd.py

```diff
@@ -7332,7 +7352,9 @@ def test_sparse_backward(self, device, dtype):
         class FixedGradientFunction(Function):
             @staticmethod
             def forward(ctx, x, grad_x):
-                ctx.save_for_backward(grad_x)
+                # this is needed because of https://github.com/pytorch/pytorch/issues/59828
```

Comment:
The fix landed now. So after rebasing on top of master, you should be able to remove this.